src/contextualbar/IContextualBarButton.ts

```diff
@@ -0,0 +1,28 @@
+import { RoomType } from '../rooms/RoomType';
+
+export interface IContextualBarButton {
```

Comment:
What would be a valid name? As `IContextualBar` is what these are being added to and the button is being registered which then the button shows the panel. The registering a button is why I called it that and because I couldn't determine a better name for it.